.devcontainer/devcontainer.json

```diff
@@ -57,11 +57,10 @@
 		"--init"
 	],
 	"settings": {
-		"terminal.integrated.defaultProfile.linux": "bash",
 		"go.toolsManagement.checkForUpdates": "local",
 		"go.useLanguageServer": true,
 		"go.gopath": "/go"
 	},
-	"workspaceFolder": "/go/src/github.com/dapr/dapr",
-	"workspaceMount": "type=bind,source=${localWorkspaceFolder},target=/go/src/github.com/dapr/dapr",
+	"workspaceFolder": "/workspaces/dapr",
```

Comment:
We have used github.com/dapr/<repo> as a standard here. Why the change?

Reply:
The devcontainer in 0.1.7 was already cloning into /workspaces - this part was left to be updated. Previously, the workspace was mounted inside the /go/src folder, but Codespaces was using /workspaces, so we were doing some hacky things to keep everything in sync.

I assume that the requirement for cloning into /go/src was due to a time when we were not using Go modules. Now that we use modules, cloning inside the GOPATH is not recommended anymore AFAIK.